pkg/proxy/endpoints.go

```diff
@@ -385,7 +422,7 @@ func (ect *EndpointChangeTracker) endpointsToEndpointsMap(endpoints *v1.Endpoint
 				}
 
 				isLocal := addr.NodeName != nil && *addr.NodeName == ect.hostname
-				baseEndpointInfo := newBaseEndpointInfo(addr.IP, int(port.Port), isLocal, nil)
+				baseEndpointInfo := newBaseEndpointInfo(addr.IP, int(port.Port), isLocal, nil, true, true, false)
```

Comment:
It could be worth adding a comment here explaining that endpoints received here will always be ready and serving and will not be terminating. This may not be obvious to someone reading through the code.